cogs/Misc.py

```diff
@@ -100,10 +119,10 @@ def dump_tables_to_csv(tables: list[Table]) -> Iterable[discord.File]:
             with io.StringIO() as f:
                 writer = csv.writer(f)
                 writer.writerow(table.column_names)
-                writer.writerows(cursor.fetchall())
+                writer.writerows(cursor)
                 f.seek(0)
                 yield discord.File(
-                    f,
+                    f,  # type: ignore (the class clearly accepts any readable, seekable IOBase)
```

Comment:
see: https://github.com/Pycord-Development/pycord/blob/master/discord/file.py#L92-L94
the type hints are buggin because this definitely works & there's no reason why a file can't exist in text mode!